plugin-management-library/src/test/java/io/jenkins/tools/pluginmanager/impl/PluginManager_removeWrapperTest.java

```diff
@@ -0,0 +1,77 @@
+package io.jenkins.tools.pluginmanager.impl;
+
+import io.jenkins.tools.pluginmanager.config.Config;
+import io.jenkins.tools.pluginmanager.config.Settings;
+import org.json.JSONObject;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class PluginManager_removeWrapperTest {
```

Comment:
Not sure if this should be done, violates JLS, I've never seen anyone doing this in test classes before?
Possibly introduce junit5 nested classes to provide the context you're looking for instead?

Reply:
My purpose is to have a separate test class with different use of @Before and such (i.e. none at all) to simplify the setup for the tests. That name with the underscore as separator is my own convention. What is the convention here about naming multiple test classes that test code in the same class under test?
It seems that Junit 4 is being used already here. I was using the principal that says to change as little as possible since I'm not familiar with what is ok to change.

Reply:
Rendered moot when those methods moved to a new class.